wave_vision/tests/config/descriptor/orb.yaml

```diff
@@ -0,0 +1,24 @@
+# Configuration parameters for ORB Descriptor Extractor
+
+# The number of points that compose the ORB descriptor. A value of 2
+# constructs the descriptor by comparing the brightnesses of a random pair
+# of points. Other values are 3 or 4; 3 will use 3 random points, and 4
+# similarly will use 4 random points. For these last two options, the
+# output result will be 2 bits, as the result is the index of the
+# brightest point. Therefore, the distance method used for the
+# BruteForceMatcher or FLANNMatcher will have to be cv::NORM_HAMMING_2, a
+# variation to the Hamming distance, when WTA_K = 3 or 4.
+#
+# Options: 2, 3, or 4.
+#
+# Default: 2.
+
+wta_k: 2
```

Comment:
Okay, renamed to `tuple_size`. It's a bit better...I got the name from [here](https://books.google.ca/books?id=LPm3DQAAQBAJ&pg=PA564&lpg=PA564&dq=ORB+WTA_K+meaning&source=bl&ots=2vHjSa7iyb&sig=Oxva0bFdtSApDtBNwNqSND79h5Y&hl=en&sa=X&ved=0ahUKEwjs5JGt2arVAhXHFT4KHVH-AmoQ6AEIXTAI#v=onepage&q=ORB%20WTA_K%20meaning&f=false).